jbpm-flow/src/main/java/org/jbpm/process/instance/context/variable/VariableScopeInstance.java

```diff
@@ -171,22 +171,25 @@ public void setContextInstanceContainer(ContextInstanceContainer contextInstance
     
     public void enforceRequiredVariables() {
         VariableScope variableScope = getVariableScope();
-        for (Variable variable : variableScope.getVariables()) {
-            String name = variable.getName();
-            if (variableScope.isRequired(name)) {  
-                // check case file if it is prefixed
-                if (name.startsWith(VariableScope.CASE_FILE_PREFIX)) {
-                    if (!findCaseData(name)) {
-                        throw new VariableViolationException(getProcessInstance().getId(), name, "Case file item '" + name + "' is required but not set");
-                        
-                    }
-                    // otherwise check variables                    
-                } else if (!hasData(variables.get(name))) {
-                    throw new VariableViolationException(getProcessInstance().getId(), name, "Variable '" + name + "' is required but not set");
-                }
-                
-            }
-        }
+		for (Variable variable : variableScope.getVariables()) {
```

Comment:
try to avoid unnecessary changes like this one. otherwise PR gets difficult to track.

Reply:
@VaniHaripriya could you undo these indentation changes as Enrique suggested?